ansible/roles/jenkins/tasks/main.yml

```diff
@@ -62,14 +62,14 @@
   when: ansible_os_family == "Debian"
   tags: jenkins
 
-- name: enable the default ssl site
-  command: a2ensite default-ssl 
-  args:
-   creates: /etc/apache2/sites-enabled/default-ssl.conf
-  when: ansible_os_family == "Debian" 
-  notify: reload apache
-  tags: jenkins
-
+# Disabled - SSL certificates need to be configured properly
```

Comment:
Sorry, I was referring to the comment "SSL certificates need to be configured properly". If we need to update the scripts to do this, then we should do it now. Otherwise, we should remove this block instead of commenting it out.